component/http/route.go

```diff
@@ -3,6 +3,7 @@ package http
 import (
 	"errors"
 	"fmt"
+	"golang.org/x/time/rate"
```

Comment:
same here with ordering

Reply:
Yep.my bad, should have revisited this PR. Will fix this. Also Do is make sense to have a check on goimports in  our make targets, like we have fmt, so that CI will fail if imports are out of order @mantzas